content/docs/guides/converting-wordpress-to-tina.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Migrating WordPress Content
+last_edited: '2024-11-08T01:49:52.552Z'
+next: ''
+previous: ''
+---
+
+## Introduction
+
+This guide is a walkthrough using a **WordPress to Markdown converter**, which is recommended for most use cases where you want to migrate your content from WordPress to Tina.
+
+This guide focuses on transferring content and media only and will not include page styling.
+
+Other conversion options that are not covered in this guide include:
+
+- Using the WordPress Rest API - best for extracting the pure html for page or post content in JSON format
+- Using the WPGraphQL plugin - best for cases where you want to query and extract specific data and content
+
+## Prerequisites
```

Comment:
from your below example, looks like you also need some react basic, you can link to there

Reply:
Done - added prerequisite to have a basic understanding of react